frontend/src/admin/components/access-rights/access-rights.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react'
+import EditableTable from '../editable-table/editable-table'
+import './access-rights.scss'
+import {User} from '../../models/user'
+
+interface AccessRightsProps {
+  selectedUser: User;
+}
+
+export default function AccessRights({ selectedUser }: AccessRightsProps) {
+  const [accessRightGroups, setAccessRightGroups] = useState(selectedUser.accessRightGroups || [])
+
+  const handleCreateGroup = () => {
```

Comment:
`handle` can be remove. You could also specify the state of the group that is created. Perhaps `createEmptyAccessRightGroup`?